app/Models/User.php

```diff
@@ -1523,17 +1524,29 @@ public function nominationModes()
 
     public function hasBlocked(self $user)
     {
-        return $this->blocks->where('user_id', $user->user_id)->count() > 0;
+        $blockIds = $this->memoize(__FUNCTION__, function () {
```

Comment:
I feel like these should be memoized as `__FUNCTION__` 🤔

Reply:
still close enough and saves me thinking of name :shrug: